assets/js/components/settings/SettingsAdmin.js

```diff
@@ -48,13 +49,21 @@ const SettingsAdmin = () => {
 
 	const { navigateTo } = useDispatch( CORE_LOCATION );
 	const goTo = ( questionIndex = 1 ) => {
+		trackEvent( 'user_input', 'settings_edit', USER_INPUT_QUESTIONS_LIST[ questionIndex - 1 ] );
```

Comment:
Maybe safer to check if that index is valid? This way we'd never run into a case where it can be `undefined`.

Reply:
👍